execution/contextTree.sh

```diff
@@ -1218,9 +1218,9 @@ function upgrade_cmdb_repo_to_v1_3_0() {
   local -A account_mappings
   readarray -t account_files < <(find "${GENERATION_DATA_DIR}" -type f -name "account.json" )
   for account_file in "${account_files[@]}"; do
-    aws_id="$( jq -r '.Account.AWSId' <"${account_file}" )"
+    provider_id="$( jq -r '.Account.ProviderId' <"${account_file}" )"
```

Comment:
Needs the check that it it is not null as per the logic below in setContext.

Looks like we would need to completely update the accounts-cmdb. Please make the logic fall back to the existing setup to give us time to sort out the transition, in line with the code in setContext.